doc/docs.md

```diff
@@ -4251,6 +4252,93 @@ fn main() {
 }
 ```
 
+## Atomics
+
+V has no special support for atomics, yet, nevertheless it's possible to treat variables as atomics
+by calling C functions from V. The standard C11 atomic functions like `atomic_store()` are usually
+defined with the help of macros and C compiler magic to provide a kind of *overloaded C functions*.
+Since V does not support overloading functions by intention there are wrapper functions defined in
+C headers named `atomic.h` that are part of the V compiler infrastructure.
+
+There are dedicated wrappers for all unsigend integer types and for pointers.
```

Comment:
`unsigned`

Reply:
Thanks, corrected. (Sometimes Emacs's Flyspell's marks should not be ignored even for keyword's...)